src/watertap_contrib/seto/unit_models/surrogate/tests/test_med_tvc_surrogate.py

```diff
@@ -0,0 +1,342 @@
+        feed = med_tvc.feed_props[0]
+        cool = med_tvc.cooling_out_props[0]
+        dist = med_tvc.distillate_props[0]
+        steam = med_tvc.heating_steam_props[0]
+        motive = med_tvc.motive_steam_props[0]
+
+        # System specification
+        # Input variable 1: Feed salinity (30-60 g/L = kg/m3)
+        feed_salinity = 35 * pyunits.kg / pyunits.m**3
+
+        # Input variable 2: Feed temperature (25-35 deg C)
+        feed_temperature = 25
+
+        # Input variable 3: Motive steam pressure (4-45 bar)
+        motive_pressure = 24
+
+        # Input variable 4: System capacity (2,000 - 100,000 m3/day)
+        sys_capacity = 2000 * pyunits.m**3 / pyunits.day
+
+        # Input variable 5: Recovery ratio (30%- 40%)
+        recovery_ratio = 0.3 * pyunits.dimensionless
+
+        feed_flow = pyunits.convert(
+            (sys_capacity / recovery_ratio), to_units=pyunits.m**3 / pyunits.s
+        )  # feed volumetric flow rate [m3/s]
+
+        """
+        Specify feed flow state properties
+        """
+        # Specify feed flow state properties
+        med_tvc.feed_props.calculate_state(
+            var_args={
+                ("flow_vol_phase", "Liq"): feed_flow,
+                ("conc_mass_phase_comp", ("Liq", "TDS")): feed_salinity,
+                ("temperature", None): feed_temperature + 273.15,
+                # feed flow is at atmospheric pressure
+                ("pressure", None): 101325,
+            },
+            hold_state=True,
+        )
+
+        """
+        Specify heating steam state properties
+        """
+        # Flow rate of liquid heating steam is zero
+        steam.flow_mass_phase_comp["Liq", "H2O"].fix(0)
+
+        # Heating steam temperature (saturated) is fixed at 70 C in this configuration
+        steam.temperature.fix(70 + 273.15)
+
+        # Calculate heating steam pressure (saturated)
+        med_tvc.heating_steam_props.calculate_state(
+            var_args={
+                ("pressure_sat", None): value(steam.pressure),
+            },
+            hold_state=True,
+        )
+        # Release vapor mass flow rate
+        steam.flow_mass_phase_comp["Vap", "H2O"].unfix()
+
+        """
+        Specify motive steam state properties
+        """
+        # Flow rate of liquid motive steam is zero
+        motive.flow_mass_phase_comp["Liq", "H2O"].fix(0)
+
+        # Calculate temperature of the motive steam (saturated)
+        med_tvc.motive_steam_props.calculate_state(
+            var_args={
+                ("pressure", None): motive_pressure * 1e5,
+                ("pressure_sat", None): motive_pressure * 1e5,
+            },
+            hold_state=True,
+        )
+        # Release vapor mass flow rate
+        motive.flow_mass_phase_comp["Vap", "H2O"].unfix()
+
+        """
+        Specify distillate flow state properties
+        """
+        # salinity in distillate is zero
+        dist.flow_mass_phase_comp["Liq", "TDS"].fix(0)
```

Comment:
same as above

Reply:
I don't disagree, but- should we be doing the same for LT-MED then? @Zhuoran29 had constraints for this in the unit model (``dist.flow_mass_phase_comp["Liq", "TDS"] == 0`` and similar for other properties) that I removed in favor of this approach. 

We could also fix these in the unit model `` build`` if we don't expect them to ever change, but I am not sure we want to be doing that sort of thing.

Reply:
Please see the comment above. The modification of LT-MED will be made in a separate PR after this one.